src/lib/db/schema.ts

```diff
@@ -109,6 +109,7 @@ export const bookings = pgTable("booking", {
   userId: text("userId")
     .notNull()
     .references(() => users.id, { onDelete: "cascade" }),
+  userType: boolean("userType").notNull(), // true if user is a member
```

Comment:
Think this column name could be more descriptive, if boolean maybe isMember? 

otherwise could be enum userType: "member" | "casual"